###Hypotheses Defination
We are testing the claim that weekly operating costs are higher than the model suggests. Let W be the true mean weekly operating cost.
1. Null Hyothesis(H0):The true mean weekly operatin cot is less than or equal to cost predicted by the model i.e H0: W<=1000 +5X
2. Alternatie Hyposthesis(H1):The true mean weekly operating cost is significantly higher than the cost predicted by the model i.e H1: W > 1000 + 5X

In [11]:
W_fixed_cost = 1000
W_variable_cost_per_unit = 5
X = 600

#Calculate the theoretical mean weekly operating cost
u = W_fixed_cost + (W_variable_cost_per_unit * X)

print("The theoretical mean weekly operating cost is: $",u)

The theoretical mean weekly operating cost is: $ 4000


In [12]:
import math

sample_std_dev_X = 25
sample_size = 25
coefficient_X =5

#calculate the standart deviation of weekly operating costs
sample_std_dev_W = coefficient_X * sample_std_dev_X

#calculate the standard error of the mean
sem = sample_std_dev_W / math.sqrt(sample_size)

print("The standard deviation of weekly operating costs is:$",sample_std_dev_W)
print("The Standard Error of the Mean is:$",sem)

The standard deviation of weekly operating costs is:$ 125
The Standard Error of the Mean is:$ 25.0


In [13]:
sample_mean = 3050

#calculate the z-test statistic
z_statistic = (sample_mean - u) / sem

print("The sample mean is:$",sample_mean)
print("The theoretical mean is:$",u)
print("The Standard Error of the Mean is:$",sem)
print("The Z-test statistic is:",z_statistic)

The sample mean is:$ 3050
The theoretical mean is:$ 4000
The Standard Error of the Mean is:$ 25.0
The Z-test statistic is: -38.0


In [14]:
from scipy.stats import norm

alpha = 0.05 # determine the critical value using give aplha
critical_z_value = norm.ppf(1 - alpha)

print("The alpha level is:",alpha)
print("The critical z -value for a one-taied test (upper tail) is:",critical_z_value)

The alpha level is: 0.05
The critical z -value for a one-taied test (upper tail) is: 1.6448536269514722


###Make a decision
we calculate the **z-statistic: -38** and  **Critical Z-value for alpha = 0.05 is 1.6448536269514722**.

Since the calculated z-statistic value -38 is not greater than the critical z-value 1.6448536269514722, we fail to reject the null hypothesis.
This means there is not enought statistical evdence at the 0.05 alpha level to conclude that the weekly operating costs of the restaurant are significantly higher than the model prediction of W=1000+5X.